docs/getting-started/installation.md

```diff
@@ -87,7 +122,7 @@ sudo port install trivy
 References:
 - <https://ports.macports.org/port/trivy/details/>
 
-### Nix/NixOS (Community)
+## Nix/NixOS (Community)
```

Comment:
The tabs formatting seems to be busted. Is it a missing dependency (add-on) in the dockerfile?

Before
<img width="548" alt="image" src="https://github.com/user-attachments/assets/1c1e6e0a-a2f0-4f5e-9952-cfcf9c959941">


After
<img width="541" alt="image" src="https://github.com/user-attachments/assets/c65cc983-1dc3-4774-b6e1-3f6a6247e86d">